app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,530 @@
+@media screen and (max-width: 375px) {
+  @import url('https://fonts.googleapis.com/css?family=Proxima+Nova');
+
+  *,
+  html {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+
+  body {
+    background-color: #f1f3f2;
+    overflow-x: hidden;
+  }
+
+  .landing-page {
+    width: 100%;
```

Comment:
- [ ] All this CSS styling doesn't apply on all pages, Please make sure that all styles are working and match the given design for all pages.

![image](https://github.com/cosywasswa/Budget-Buddy/assets/68796232/039cddf5-0b94-4244-b6e2-e1930f4dc529)

![image](https://github.com/cosywasswa/Budget-Buddy/assets/68796232/0c65069e-3cb1-4c24-b59c-420fbd29f36b)